docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
I don't see `images/gardenerazuresecret.jpg` being linked/used by this tutorial. Can we remove it or make the tutorial use it?